tests/SessionTest.cs

```diff
@@ -1,6 +1,9 @@
 ﻿using System;
 using AeroGear.OAuth2;
 using Microsoft.VisualStudio.TestPlatform.UnitTestFramework;
```

Comment:
Tried to compile it with mono, but there are just to many system dependent things in this project. Benefit of using this is that the test will run on the simulator.